### DISCUSSION TITLE
How to use nautilus as the file manager?

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6590] [Q&A] [answered]
How can I use nautilus for the `c.fileselect.multiple_files.command` and
`c.fileselect.single_file.command` ?  Sorry but I haven't figured it out.

If it is not possible with Nautilus, then If someone can suggest a GUI file
manager that is not from KDE and works with qutebrowser I'll be thankfull

**Comment by The-Compiler** [comment, marked as answer]
I don't think this is possible. You'd need a file manager which supports selecting a path and writing that path to a file, which AFAIK only CLI-filemanagers do.

Though if you're on GTK, Qt should use the GTK file browser already, which is based on the same code as Nautilus IIRC? What's your goal?

**Reply by uwla** [reply]
Qt uses GTK 2.0 file browser. The thing is that I'd like to have some bookmark
folders for quick access when downloading/uploading files and AFAIK Qt's
built-in GTK file browser doesn't do that. I wanted to use Nautilus because it
is very beautiful.

Anyway, seems like I'll have to use a terminal file browser to have bookmarks.
Do you know one which supports selecting files with the mouse?  Terminal/cli
programms are good to automate stuff but usually when I upload or download
files there isn't anything to automate, thus using mouse is more productive in
those cases. A terminal browser with mouse selection support would be good.

Thanks

**Reply by uwla** [reply]
Actually, I just found out that ranger supports mouse. So, although I think nautilus is prettier,
ranger is good enough for my purpose
